<a href="https://colab.research.google.com/github/HyungSang8421/-/blob/main/Word_Fast_Doc.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# 환경변수 설정 및 라이브러리 설치

kopora - 말뭉치 제공 라이브러리   / 코포라 사이트 (https://ko-nlp.github.io/Korpora/ko-docs/introduction/)
  
gesim - word2vec,fasttext,doc2vec 라이브러리 제공

# Word2vec 실습

In [1]:
!pip install  korpora
!pip install  gensim
!apt-get install openjdk-20-jdk# JVM 설치


     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.8/57.8 kB 1.0 MB/s eta 0:00:00


Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package openjdk-20-jdk#
E: Unable to locate package JVM
E: Unable to locate package 설치


Word2vec(CBOW실습) 5~7분소요

In [2]:
from gensim.models import Word2Vec
from Korpora import Korpora

# "korean_petitions" 데이터셋 로드
corpus = Korpora.load("korean_petitions")

# 텍스트 전처리: 문장을 단어로 분할
sentences = [s for doc in corpus.get_all_texts() for s in doc.split('. ')]  # 문서를 문장으로 분할

# 단어로 분할된 문장 리스트
word_sentences = [sentence.split() for sentence in sentences]

# Word2Vec 모델 학습 (CBOW)
cbow_model = Word2Vec(word_sentences, vector_size=100, window=5, min_count=5, workers=4, sg=0)

# 단어 벡터 출력
word_vectors_cbow = cbow_model.wv

# 검색할 단어
target_word = "가상화폐"
print("CBOW - '{}'의 벡터:".format(target_word), word_vectors_cbow[target_word])
# 유사한 단어 찾기
similar_words = word_vectors_cbow.most_similar(target_word)

# 검색한 단어가 유사한 단어 목록에 포함되는 경우 제외한 유사한 단어 목록 생성
filtered_similar_words = [(word, score) for word, score in similar_words if word != target_word and target_word not in word]

print("검색한 단어:", target_word)
print("유사한 단어들:", filtered_similar_words)



    Korpora 는 다른 분들이 연구 목적으로 공유해주신 말뭉치들을
    손쉽게 다운로드, 사용할 수 있는 기능만을 제공합니다.

    말뭉치들을 공유해 주신 분들에게 감사드리며, 각 말뭉치 별 설명과 라이센스를 공유 드립니다.
    해당 말뭉치에 대해 자세히 알고 싶으신 분은 아래의 description 을 참고,
    해당 말뭉치를 연구/상용의 목적으로 이용하실 때에는 아래의 라이센스를 참고해 주시기 바랍니다.

    # Description
    Author : Hyunjoong Kim lovit@github
    Repository : https://github.com/lovit/petitions_archive
    References :

    청와대 국민청원 게시판의 데이터를 월별로 수집한 것입니다.
    청원은 게시판에 글을 올린 뒤, 한달 간 청원이 진행됩니다.
    수집되는 데이터는 청원종료가 된 이후의 데이터이며, 청원 내 댓글은 수집되지 않습니다.
    단 청원의 동의 개수는 수집됩니다.
    자세한 내용은 위의 repository를 참고하세요.

    # License
    CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
    Details in https://creativecommons.org/publicdomain/zero/1.0/



[korean_petitions] download petitions_2017-08: 1.84MB [00:00, 17.3MB/s]                            
[korean_petitions] download petitions_2017-09: 20.4MB [00:00, 102MB/s]                             
[korean_petitions] download petitions_2017-10: 12.0MB [00:00, 75.7MB/s]                            
[korean_petitions] download petitions_2017-11: 28.4MB [00:00, 116MB/s]                            
[korean_petitions] download petitions_2017-12: 29.0MB [00:00, 133MB/s]                            
[korean_petitions] download petitions_2018-01: 43.9MB [00:00, 153MB/s]                            
[korean_petitions] download petitions_2018-02: 33.8MB [00:00, 117MB/s]                            
[korean_petitions] download petitions_2018-03: 34.3MB [00:00, 118MB/s]                            
[korean_petitions] download petitions_2018-04: 35.5MB [00:01, 35.1MB/s]                            
[korean_petitions] download petitions_2018-05: 37.5MB [00:00, 105MB/s]                            
[korea

검색한 단어: 가상화폐
유사한 단어들: [('암호화폐', 0.8845651745796204), ('거래소', 0.8571281433105469), ('비트코인', 0.8427028059959412), ('코인', 0.8054822087287903), ('가상화페', 0.7825321555137634), ('주식', 0.7699881196022034), ('코스닥', 0.7295181155204773), ('거래소를', 0.7170189619064331), ('공매도', 0.7151204943656921)]




Skip-gram 실습 20~24분소요

In [1]:
from gensim.models import Word2Vec
from Korpora import Korpora

# "korean_petitions" 데이터셋 로드
corpus = Korpora.load("korean_petitions")

# 텍스트 전처리: 문장을 단어로 분할
sentences = [s for doc in corpus.get_all_texts() for s in doc.split('. ')]  # 문서를 문장으로 분할

# 단어로 분할된 문장 리스트
word_sentences = [sentence.split() for sentence in sentences]

# Word2Vec 모델 학습 (Skip-Gram)
skipgram_model = Word2Vec(word_sentences, vector_size=100, window=5, min_count=5, workers=4, sg=1)

# 단어 벡터 출력
word_vectors_skipgram = skipgram_model.wv

# "국민"이라는 단어의 벡터 출력
target_word = "가상화폐"
print("Skip-Gram - '{}'의 벡터:".format(target_word), word_vectors_skipgram[target_word])

# 유사한 단어 찾기
similar_words = word_vectors_skipgram.most_similar(target_word)

# 검색한 단어가 유사한 단어 목록에 포함되는 경우 제외한 유사한 단어 목록 생성
filtered_similar_words = [(word, score) for word, score in similar_words if word != target_word and target_word not in word]

print("검색한 단어:", target_word)
print("유사한 단어들:", filtered_similar_words)




    Korpora 는 다른 분들이 연구 목적으로 공유해주신 말뭉치들을
    손쉽게 다운로드, 사용할 수 있는 기능만을 제공합니다.

    말뭉치들을 공유해 주신 분들에게 감사드리며, 각 말뭉치 별 설명과 라이센스를 공유 드립니다.
    해당 말뭉치에 대해 자세히 알고 싶으신 분은 아래의 description 을 참고,
    해당 말뭉치를 연구/상용의 목적으로 이용하실 때에는 아래의 라이센스를 참고해 주시기 바랍니다.

    # Description
    Author : Hyunjoong Kim lovit@github
    Repository : https://github.com/lovit/petitions_archive
    References :

    청와대 국민청원 게시판의 데이터를 월별로 수집한 것입니다.
    청원은 게시판에 글을 올린 뒤, 한달 간 청원이 진행됩니다.
    수집되는 데이터는 청원종료가 된 이후의 데이터이며, 청원 내 댓글은 수집되지 않습니다.
    단 청원의 동의 개수는 수집됩니다.
    자세한 내용은 위의 repository를 참고하세요.

    # License
    CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
    Details in https://creativecommons.org/publicdomain/zero/1.0/

[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-08
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-09
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/kore

# FastText 실습

Fasttext(CBOW)실습 15분소요

In [1]:
from gensim.models import FastText
from Korpora import Korpora
import re

# "korean_petitions" 데이터셋 로드
corpus = Korpora.load("korean_petitions")

# 텍스트 전처리: 문장을 단어로 분할
sentences = [s for doc in corpus.get_all_texts() for s in doc.split('. ')]  # 문서를 문장으로 분할

# 한글이 아닌 글자 제외하고 전처리
korean_sentences = [re.sub("[^가-힣a-zA-Z ]", "", sentence) for sentence in sentences]

# 단어로 분할된 문장 리스트
word_sentences = [sentence.split() for sentence in korean_sentences]

# FastText 모델 학습
fasttext_model = FastText(word_sentences, vector_size=100, window=5, min_count=5, workers=4, sg=0)

# 단어 벡터 출력
word_vectors_fasttext = fasttext_model.wv

# 검색할 단어
target_word = "가상화폐"

# 조사 제거를 위한 함수
def remove_josa(word):
    josa_list = ['은', '는', '이', '가', '을', '를', '와', '과']
    for josa in josa_list:
        if word.endswith(josa):
            return word[:-len(josa)]
    return word

# 검색할 단어에 조사 제거
target_word_no_josa = remove_josa(target_word)

print("FastText - '{}'의 벡터:".format(target_word_no_josa), word_vectors_fasttext[target_word_no_josa])

# 유사한 단어 찾기
similar_words = word_vectors_fasttext.most_similar(target_word_no_josa)

print("검색한 단어:", target_word)
print("유사한 단어들:", similar_words)


    Korpora 는 다른 분들이 연구 목적으로 공유해주신 말뭉치들을
    손쉽게 다운로드, 사용할 수 있는 기능만을 제공합니다.

    말뭉치들을 공유해 주신 분들에게 감사드리며, 각 말뭉치 별 설명과 라이센스를 공유 드립니다.
    해당 말뭉치에 대해 자세히 알고 싶으신 분은 아래의 description 을 참고,
    해당 말뭉치를 연구/상용의 목적으로 이용하실 때에는 아래의 라이센스를 참고해 주시기 바랍니다.

    # Description
    Author : Hyunjoong Kim lovit@github
    Repository : https://github.com/lovit/petitions_archive
    References :

    청와대 국민청원 게시판의 데이터를 월별로 수집한 것입니다.
    청원은 게시판에 글을 올린 뒤, 한달 간 청원이 진행됩니다.
    수집되는 데이터는 청원종료가 된 이후의 데이터이며, 청원 내 댓글은 수집되지 않습니다.
    단 청원의 동의 개수는 수집됩니다.
    자세한 내용은 위의 repository를 참고하세요.

    # License
    CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
    Details in https://creativecommons.org/publicdomain/zero/1.0/

[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-08
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-09
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/kore

Fasttext(Skip-gram)실습 14~17분소요

In [1]:
from gensim.models import FastText
from Korpora import Korpora
import re

# "korean_petitions" 데이터셋 로드
corpus = Korpora.load("korean_petitions")

# 텍스트 전처리: 문장을 단어로 분할
sentences = [s for doc in corpus.get_all_texts() for s in doc.split('. ')]  # 문서를 문장으로 분할

# 한글이 아닌 글자 제외하고 전처리
korean_sentences = [re.sub("[^가-힣a-zA-Z ]", "", sentence) for sentence in sentences]

# 단어로 분할된 문장 리스트
word_sentences = [sentence.split() for sentence in korean_sentences]

# FastText 모델 학습
fasttext_model = FastText(word_sentences, vector_size=100, window=5, min_count=5, workers=4, sg=1)

# 검색할 단어
target_word = "가상화폐"

# 조사 제거를 위한 함수
def remove_josa(word):
    josa_list = ['은', '는', '이', '가', '을', '를', '와', '과']
    for josa in josa_list:
        if word.endswith(josa):
            return word[:-len(josa)]
    return word

# 검색할 단어에 조사 제거
target_word_no_josa = remove_josa(target_word)

print("FastText - '{}'의 벡터:".format(target_word_no_josa), fasttext_model.wv[target_word_no_josa])

# 유사한 단어 찾기
similar_words = fasttext_model.wv.most_similar(target_word_no_josa)

print("검색한 단어:", target_word)
print("유사한 단어들:", similar_words)



    Korpora 는 다른 분들이 연구 목적으로 공유해주신 말뭉치들을
    손쉽게 다운로드, 사용할 수 있는 기능만을 제공합니다.

    말뭉치들을 공유해 주신 분들에게 감사드리며, 각 말뭉치 별 설명과 라이센스를 공유 드립니다.
    해당 말뭉치에 대해 자세히 알고 싶으신 분은 아래의 description 을 참고,
    해당 말뭉치를 연구/상용의 목적으로 이용하실 때에는 아래의 라이센스를 참고해 주시기 바랍니다.

    # Description
    Author : Hyunjoong Kim lovit@github
    Repository : https://github.com/lovit/petitions_archive
    References :

    청와대 국민청원 게시판의 데이터를 월별로 수집한 것입니다.
    청원은 게시판에 글을 올린 뒤, 한달 간 청원이 진행됩니다.
    수집되는 데이터는 청원종료가 된 이후의 데이터이며, 청원 내 댓글은 수집되지 않습니다.
    단 청원의 동의 개수는 수집됩니다.
    자세한 내용은 위의 repository를 참고하세요.

    # License
    CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
    Details in https://creativecommons.org/publicdomain/zero/1.0/

[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-08
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/korean_petitions/petitions_2017-09
[Korpora] Corpus `korean_petitions` is already installed at /root/Korpora/kore

# Doc2vec 실습

Doc2vec(DBOW)실습 25분소요

In [ ]:
from gensim.models import Doc2Vec
from gensim.models.doc2vec import TaggedDocument
from Korpora import Korpora
import re

# Korpora 라이브러리를 통해 한국어 청원 데이터를 불러옵니다
corpus = Korpora.load("korean_petitions")
petitions = corpus.get_all_texts()

# 전처리 함수 정의: 한글 아닌 글자 제거
def preprocess_text(text):
    text = re.sub("[^가-힣a-zA-Z ]", "", text)  # 한글 이외의 글자 제거
    text = text.strip()  # 문서 양 끝 공백 제거
    return text

# 전처리 적용한 데이터로 TaggedDocument 생성
tagged_data = [TaggedDocument(words=preprocess_text(petition).split(), tags=[str(i)]) for i, petition in enumerate(petitions)]

# DBOW 방식의 Doc2Vec 모델을 초기화하고 학습시킵니다
dbow_model = Doc2Vec(vector_size=100, window=5, min_count=1, workers=4, dm=0, epochs=20)
dbow_model.build_vocab(tagged_data)
dbow_model.train(tagged_data, total_examples=dbow_model.corpus_count, epochs=dbow_model.epochs)

# 특정 문서의 인덱스를 지정합니다
target_doc_index = 0

# DBOW 방식으로 얻은 특정 문서의 벡터 출력
doc_vector_dbow = dbow_model.dv[target_doc_index]
print("DBOW 방식으로 얻은 특정 문서의 벡터:", doc_vector_dbow)

# 특정 문서의 내용 출력
print("\n특정 문서 내용:")
print(petitions[target_doc_index])
print("\n")

# 특정 문서와 유사한 문서를 찾습니다
similar_documents_dbow = dbow_model.dv.most_similar(target_doc_index, topn=5)

print("특정 문서와 유사한 문서 (DBOW 방식):")
for doc_id, score in similar_documents_dbow:
    print(f"유사도 {score:.2f}, 문서 인덱스 {doc_id}:")
    print(petitions[int(doc_id)])
    print("\n")


Doc2vec(DM 실습) 35분 소요

In [ ]:
from gensim.models import Doc2Vec
from gensim.models.doc2vec import TaggedDocument
from Korpora import Korpora
import re

# Korpora 라이브러리를 통해 한국어 청원 데이터를 불러옵니다
corpus = Korpora.load("korean_petitions")
petitions = corpus.get_all_texts()

# 전처리 함수 정의: 한글 아닌 글자 제거
def preprocess_text(text):
    text = re.sub("[^가-힣a-zA-Z ]", "", text)  # 한글 이외의 글자 제거
    text = text.strip()  # 문서 양 끝 공백 제거
    return text

# 전처리 적용한 데이터로 TaggedDocument 생성
tagged_data = [TaggedDocument(words=preprocess_text(petition).split(), tags=[str(i)]) for i, petition in enumerate(petitions)]

# DM 방식의 Doc2Vec 모델을 초기화하고 학습시킵니다
dm_model = Doc2Vec(vector_size=100, window=5, min_count=1, workers=4, dm=1, epochs=20)
dm_model.build_vocab(tagged_data)
dm_model.train(tagged_data, total_examples=dm_model.corpus_count, epochs=dm_model.epochs)

# 특정 문서의 인덱스를 지정합니다
target_doc_index = 0

# DM 방식으로 얻은 특정 문서의 벡터 출력
doc_vector_dm = dm_model.dv[target_doc_index]
print("DM 방식으로 얻은 특정 문서의 벡터:", doc_vector_dm)

# 특정 문서의 내용 출력
print("\n특정 문서 내용:")
print(petitions[target_doc_index])
print("\n")

# 특정 문서와 유사한 문서를 찾습니다
similar_documents_dm = dm_model.dv.most_similar(target_doc_index, topn=5)

print("특정 문서와 유사한 문서 (DM 방식):")
for doc_id, score in similar_documents_dm:
    print(f"유사도 {score:.2f}, 문서 인덱스 {doc_id}:")
    print(petitions[int(doc_id)])
    print("\n")


# 결과 값 예시

# Word2vec,FastText 키워드 가상화폐 / Doc2vec 인덱스 0

[Word2Vec]    
----------------------------------------------------------------------------
  
----------------------------------------------------------------------------
CBOW  
검색한 단어: 가상화폐
유사한 단어들: [('암호화폐', 0.882172167301178), ('거래소', 0.8729402422904968), ('비트코인', 0.8472152948379517), ('코인', 0.7633044719696045), ('주식', 0.7607668042182922), ('코스닥', 0.75547194480896), ('가상화페', 0.7466930747032166), ('공매도', 0.722719132900238), ('거래소를', 0.7169374823570251)]
  
---------------------------------------------------------------------------
  
Skip gram  
검색한 단어: 가상화폐
유사한 단어들: [('암호화폐', 0.9376102685928345), ('거래소', 0.927099347114563), ('비트코인', 0.8701239824295044), ('가상화페', 0.8493988513946533), ('거래소를', 0.8490596413612366), ('거래소에', 0.8220364451408386), ('코인', 0.816824197769165), ('주식', 0.8152421116828918), ('가상', 0.8056983351707458)]
  
---------------------------------------------------------------------------
  
[FastText]  
-------------------------------------------------------------------------
  
-------------------------------------------------------------------------
CBOW  
검색한 단어: 가상화폐
유사한 단어들: [('가상화폐폐지', 0.9647992849349976), ('가상화폐규제', 0.9616738557815552), ('가상화페만', 0.9601261019706726), ('가상화패', 0.9596997499465942), ('가상화페와', 0.9594783186912537), ('가상화패도', 0.9593979120254517), ('가상화폐등', 0.9591593742370605), ('가상화페로', 0.9586182832717896), ('가상화폐나', 0.9583812355995178), ('가상화페도', 0.957714855670929)]  
  
----------------------------------------------------------------------------
  
skip gram  
검색한 단어: 가상화폐
유사한 단어들: [('가상화폐암호화폐', 0.9495850801467896), ('암호화폐', 0.9467371702194214), ('가상화폐든', 0.9465683698654175), ('가상화폐고', 0.9449959993362427), ('가상화폐등', 0.9430037140846252), ('가상화폐폐지', 0.9418099522590637), ('가상화폐을', 0.9410449862480164), ('가상화폐니', 0.9389110803604126), ('가상화폐야', 0.9349937438964844), ('가상화페와', 0.9345189929008484)]   
    
---------------------------------------------------------------------------
[Doc2vec]  
---------------------------------------------------------------------------
   
--------------------------------------------------------------------------

 DBOW    

특정 문서 내용:
안녕하세요. 현재 사대, 교대 등 교원양성학교들의 예비교사들이 임용절벽에 매우 힘들어 하고 있는 줄로 압니다. 정부 부처에서는 영양사의 영양'교사'화, 폭발적인 영양'교사' 채용, 기간제 교사, 영전강, 스강의 무기계약직화가 그들의 임용 절벽과는 전혀 무관한 일이라고 주장하고 있지만 조금만 생각해보면 전혀 설득력 없는 말이라고 생각합니다. 학교 수가 같고, 학생 수가 동일한데 영양교사와 기간제 교사, 영전강 스강이 학교에 늘어나게 되면 당연히 정규 교원의 수는 줄어들게 되지 않겠습니까? 기간제 교사, 영전강, 스강의 무기계약직화, 정규직화 꼭 전면 백지화해주십시오. 백년대계인 국가의 교육에 달린 문제입니다. 단순히 대통령님의 일자리 공약, 81만개 일자리 창출 공약을 지키시고자 돌이킬 수 없는 실수는 하지 않으시길 바랍니다. 세계 어느 나라와 비교해도, 한국 교원의 수준과 질은 최고 수준입니다. 고등교육을 받고 어려운 국가 고시를 통과해야만 대한민국 공립 학교의 교단에 설 수 있고, 이러한 과정이 힘들기는 하지만 교원들이 교육자로서의 사명감과 자부심을 갖고 교육하게 되는 원동력이기도 합니다. 자격도 없는 비정규 인력들을 일자리 늘리기 명목 하에 학교로 들이게 되면, 그들이 무슨 낯으로 대한민국이 '공정한 사회' 라고 아이들에게 가르칠 수 있겠습니까? 그들이 가르치는 것을 학부모와 학생들이 납득할 수 있겠으며, 학생들은 공부를 열심히 해야하는 이유를 찾을 수나 있겠습니까? 열심히 안 해도 떼 쓰면 되는 세상이라고 생각하지 않겠습니까? 영양사의 영양교사화도 재고해주십시오. 영양사분들 정말 너무나 고마운 분들입니다. 학생들의 건강과 영양? 당연히 성장기에 있는 아이들에게 필수적이고 중요한 문제입니다. 하지만 이들이 왜 교사입니까. 유래를 찾아 볼 수 없는 영양사의 '교사'화. 정말 대통령님이 생각하신 아이디어라고 믿기 싫을 정도로 납득하기 어렵습니다. 중등은 실과교과 교사가 존재하지요? 초등 역시 임용 시험에 실과가 포함돼 있으며 학교 현장에서도 정규 교원이 직접 실과 과목을 학생들에게 가르칩니다. 영양'교사', 아니 영양사가 학생들에게 실과를 가르치지 않습니다. 아니 그 어떤 것도 가르치지 않습니다. 올해 대통령님 취임 후에 초등, 중등 임용 티오가 초전박살 나는 동안 영양'교사' 티오는 폭발적으로 확대된 줄로 압니다. 학생들의 교육을 위해 정말 교원의 수를 줄이고, 영양 교사의 수를 늘리는 것이 올바른 해답인지 묻고 싶습니다. 마지막으로 교원 당 학생 수. 이 통계도 제대로 내주시기 바랍니다. 다른 나라들은 '정규 교원', 즉 담임이나 교과 교사들로만 통계를 내는데(너무나 당연한 것이지요) 왜 한국은 보건, 영양, 기간제, 영전강, 스강 까지 다 포함해서 교원 수 통계를 내는건가요? 이런 통계의 장난을 통해 OECD 평균 교원 당 학생 수와 거의 비슷한 수준에 이르렀다고 주장하시는건가요? 학교는 교육의 장이고 학생들의 공간이지, 인력 센터가 아닙니다. 부탁드립니다. 부디 넓은 안목으로 멀리 내다봐주시길 간곡히 부탁드립니다.
  

특정 문서와 유사한 문서 (DBOW 방식):
유사도 0.81, 문서 인덱스 144764:
학교 현장에 실력없는 선생님들이 너무 많습니다. 학생평가를 통해 적정 점수에 미달된 교사들을 대상으로 재평가를 진행해주세요. 임용의 수는 해마다 줄고 있는데 실력없는 교사들로인하여 새로운 자리가 나질 않고 있으니 실력있는 교사들을 새로 뽑는 것이 일자리 창출과 교육현장 모두에게 좋은 일이에요.
  

유사도 0.79, 문서 인덱스 278982:
지금 대통령님 곁에는 교육 현장 경험이 있는 사람이 한 명도 없습니까? 교육은 백년지대계라고 하며, 긴 시간을 두고 점진적으로 이루어 지는 것이 교육이고, 사람을 직접 교육한다는 점에서 다른 정부 기관보다도 더 장관 지명에 신중해야 된다고 생각합니다. 하지만 현 장관 후보자는 현장 경험이 전무한 분으로 단지 국회에서 교육 관련 상임위원회 경력이 전부입니다. 아무리 훌륭한 정책이라도 현장에 있는 학교에 적용해 보면 이론만 훌륭할 뿐이지 현장과는 동떨어진 경우가 너무나 많습니다. 교직에 있는 국민 한 사람으로써 이런 정책을 볼 때마다 참으로 안타까움을 느낍니다. 현장과 이론은 너무도 다릅니다. 왜냐하면 상대가 사람이기 때문입니다. 사람은 다른 사물과 달라서 각자가 사고를 하고, 이 높은 수준의 사고를 통해 행동하며 인간만의 문화를 만들어 갑니다. 따라서 단순한 이론을 가지고는 이를 교육 현장에 투입하는 것은 지극히 위험한 것입니다. 따라서 현장 경험은 그 어떤 분야 보다도 필요하고 중요한 것이지요.. 저는 유은혜 교육부장관 지명자의 능력이 부족하다거나, 인성이 부족하다고 생각하지 않습니다. 훌륭한 분이라고 생각하고 평소에도 존경해 왔습니다. 하지만 안타깝게도 교육 현장의 경험은 없습니다! 대통령님의 곁에 단 1명도 교육 현장 경험이 있는 분이 없다고 생각하지는 않습니다...그 분(지명되지 못한 분)에게 또 다른 문제(?)가 검증 과정에서 나왔을 수 있습니다...대학교, 중등학교, 초등학교, 유아교육 등 교육 현장에서 종사하는 많은 사람들이 있습니다...진정 이나라를 위하신다면 좀 늦더라도 각계 각층의 추천을 받아 진실된 검증을 통해 현장 경험이 있는 분을 수천년을 이어나갈 이 자랑스런 대한민국의 교육 수장으로 모셨으면 합니다...
  

유사도 0.79, 문서 인덱스 144684:
-본 내용과 비슷한 내용의 청원이 있었지만, 덧붙여 얘기하고 싶은 말이 있어서 다시 한 번 얘기합니다. 이 청원에 동의하시는 분들이 계시다면, https://www1.president.go.kr/petitions/175020 이 링크의 청원도 동의해주시기 바랍니다. 아래에도 링크 추가합니다. 수능 최저 등급 폐지는 학생들의 부담을 덜어주기는커녕 기존 수시 제도의 단점만 부각시킬 것입니다. 저도 현재 입시를 준비하는 고등학교 3학년으로서 부족한 글쓰기로나마 제가 생각하는 이유 몇 줄 써봅니다. 첫째, 객관적 실력 평가가 어렵습니다. 학생부 전형에서 가장 크게 보는 것은 교내 내신 성적입니다. 앞으로 고교 평준화를 한다고는 하지만 학교 간 어쩔 수 없는 실력, 분위기 등 차이는 존재합니다. 전국 단위로 가장 객관적이라 할 수 있는 수능 시험을 수시 입시에 반영하지 않는다면 대학에서도 지원 학생들 모두에게 적용되는 객관적이고 절대적인 평가 기준을 잡기 힘들 것입니다. 둘째, 최저등급을 폐지한다고 해도 학생들의 부담이 아주 크게 덜어지는 것은 아닙니다. 대입에 보장되고 확실한 것이 어디 있습니까? 학생들이 아무리 수시 전형에 주력한다고 해도, 수시로 지원한 모든 학생들이 원하는 대학, 아니 최소한 수시로 대학에 합격할 수 있는 것은 아닙니다. 그것은 아무리 내신 성적이 좋은 학생이라도 마찬가지입니다. 따라서 어떤 학생이라도 수능 공부를 완전히 제쳐 둘 수는 없습니다. 만약을 대비해야 하기 때문입니다. 셋째, 이런 식으로 대입에서 수능의 비율을 축소하고 수시 제도를 늘리기에는 현재의 수시 제도가 수능보다 월등히 뛰어나다고 볼 수 없습니다. 위에서 언급한 객관성 측면도 있지만, 다른 문제가 많습니다. 우선 내신 성적은 각 학교, 각 반 안의 학생들을 경쟁시키고, 그들을 순서대로 줄 세워 내는 것입니다. 바로 옆 짝꿍과 경쟁을 해야만 하는 것입니다. 이러한 상황에서 학생들끼리 교류하고, 토론하고, 함께 성장하는 분위기가 전국 고교에 자리잡을 수 있을까요? 저는 어렵다고 봅니다. 학생부도 문제가 많습니다. 생활부는 일단은 선생님들께서 학생들을 관찰하고 기록하는 것인데, 현재 한 학급에 학생이 30~40명, 한 학년에 학생이 몇 백 명 되면서도 한 선생님께서 그 학생들을 모두 가르치는 구조로는 절대로, 현재 학생부가 좋아서 좋은 대학교에 가시는 분들의 생활부, 그러니까 바람직한 생활부가 나올 수 없습니다. 담임 선생님이 아닌 교과 선생님의 경우 한 학년 전체의 수업에 들어가는데, 어떻게 어떤 학생이 무엇을 했으며, 또 그 학생이 그 활동으로 인해 어떤 가치를 배웠는지 알고 기록할 수 있을까요? 이 외에도 학생부 종합전형으로 대표되는 수시 제도는 문제점이 많습니다. 수능이 문제가 없다는 것이 아니라, 정부에서 각 대학교에 수시를 대폭 늘리고, 수능 최저 등급제까지 없애라고 권고하기에는 수시도 절대 완벽한 전형이 아니라는 것입니다. 정말로 학생들의 부담을 줄여 주고 싶으시다면, 장기적으로라도 근본적인 제도 개혁을 검토해주십시오. 제가 읽은 교육 관련 책에서 우리나라 학생들은, 세상에서 제일 재미 없는 공부를 세상에서 제일 오랫동안 하고 있다고 합니다. 이게 재미만 없으면 다행입니다. 특히 입시에 당면한 고3 학생들은 매일매일 불안한 미래를 생각하며 반복에서 지식을을 머릿속에 우겨넣고 있습니다. 교육부는 부디, 현재 교육 방향, 내신 제도, 대학 제도가 미래의 사회 구성원들을 자라게 위한 것인지, 단순히 대학 보내고 기업에 취업시켜서 어떻게든 기존의 사회 구조가 돌아가게 할 노동력만을 생산하기 위한 것인지 생각해주십시오. 시간이 얼마나 걸려도 좋습니다. 17~19세의 청소년들이 극한의 스트레스를 매일 받아가며 생활하는 것이, 이전부터 해왔다는 이유만으로 지속해야 할 것인지 제발 고려해주십시오. 부족하고 긴 글 읽어 주셔서 감사합니다.

  
유사도 0.78, 문서 인덱스 73614:
우리나라에서 가장 바뀌기 힘든 부분중의 하나가 교육분야가 아닐까 생각합니다. 그리고 많은 기득권층을 가지고 있는 분야는 더욱 그러하다고 생각합니다. 우리나라의 교육정책들은 미래를 위해 나아가고 있지만 교사 임용 방식은 구시대적인 것 같습니다. 교대나 사범대 학생들을 성적순으로 뽑고 그들만 교사 임용고시를 보게되니, 자질 문제를 거론할수 밖에 없는 것 같습니다. 교대나 사범대를 나오면 모두가 아이들을 잘 가르칠수 있는지 의문이 됩니다. 20살 남짓되는 교대, 사범대 지원자들 모두가 교사의 자질이 충분하여 뽑히는 것인지요. 교사 임용고시는 너무나 제한된 범위내에서 교사 자질이 없는 사람들이 포함되어 치러지는 것 같아 아쉬움이 큽니다. 일부 국가에서는 교사의 자질을 교대나 사범대나 임용고시로 판단하지 않고 전문성을 보는 것으로 알고 있습니다. 로스쿨이나 의학전문대학원이 이를 반영한다고 생각합니다. 물이 고이면 반드시 썩기 마련입니다. 선진 대한민국을 위해 가장 필요한 것은 인재 양성이 아닐까요. 이를 뒷받침하는 교사의 임용 정책이 반드시 바뀌어서 참교육인들이 아이들을 가르쳐주었으면 좋겠습니다.
  

유사도 0.78, 문서 인덱스 151773:
대한민국 학생들은 다른 OECD 국가와 달리 주당 50시간 정도를 공부하고 있습니다. 이는 OECD 평균보다 15~20시간 정도 더 많이 공부하는 수치입니다. 1년을 기준으로 했을 때 대한민국의 고등학생들은 1년에 약 3,000시간을 공부합니다. OECD 국가 중 하나인 네덜란드와 대조를 한다면 대한민국의 학생이 3배 이상을 1년간 공부를 합니다. 한국의 입시전형 때문에 그렇다는 것은 잘 알지만 자라나는 학생들에게 1년에 3,000시간은 너무 가혹하다고 생각합니다. 그러니, 저를 비롯한 학생들의 연간 공부시간을 축소할 것을 진심으로 요구합니다.

  
------------------------------------------------------------------------  
DM  

    
특정 문서 내용:
안녕하세요. 현재 사대, 교대 등 교원양성학교들의 예비교사들이 임용절벽에 매우 힘들어 하고 있는 줄로 압니다. 정부 부처에서는 영양사의 영양'교사'화, 폭발적인 영양'교사' 채용, 기간제 교사, 영전강, 스강의 무기계약직화가 그들의 임용 절벽과는 전혀 무관한 일이라고 주장하고 있지만 조금만 생각해보면 전혀 설득력 없는 말이라고 생각합니다. 학교 수가 같고, 학생 수가 동일한데 영양교사와 기간제 교사, 영전강 스강이 학교에 늘어나게 되면 당연히 정규 교원의 수는 줄어들게 되지 않겠습니까? 기간제 교사, 영전강, 스강의 무기계약직화, 정규직화 꼭 전면 백지화해주십시오. 백년대계인 국가의 교육에 달린 문제입니다. 단순히 대통령님의 일자리 공약, 81만개 일자리 창출 공약을 지키시고자 돌이킬 수 없는 실수는 하지 않으시길 바랍니다. 세계 어느 나라와 비교해도, 한국 교원의 수준과 질은 최고 수준입니다. 고등교육을 받고 어려운 국가 고시를 통과해야만 대한민국 공립 학교의 교단에 설 수 있고, 이러한 과정이 힘들기는 하지만 교원들이 교육자로서의 사명감과 자부심을 갖고 교육하게 되는 원동력이기도 합니다. 자격도 없는 비정규 인력들을 일자리 늘리기 명목 하에 학교로 들이게 되면, 그들이 무슨 낯으로 대한민국이 '공정한 사회' 라고 아이들에게 가르칠 수 있겠습니까? 그들이 가르치는 것을 학부모와 학생들이 납득할 수 있겠으며, 학생들은 공부를 열심히 해야하는 이유를 찾을 수나 있겠습니까? 열심히 안 해도 떼 쓰면 되는 세상이라고 생각하지 않겠습니까? 영양사의 영양교사화도 재고해주십시오. 영양사분들 정말 너무나 고마운 분들입니다. 학생들의 건강과 영양? 당연히 성장기에 있는 아이들에게 필수적이고 중요한 문제입니다. 하지만 이들이 왜 교사입니까. 유래를 찾아 볼 수 없는 영양사의 '교사'화. 정말 대통령님이 생각하신 아이디어라고 믿기 싫을 정도로 납득하기 어렵습니다. 중등은 실과교과 교사가 존재하지요? 초등 역시 임용 시험에 실과가 포함돼 있으며 학교 현장에서도 정규 교원이 직접 실과 과목을 학생들에게 가르칩니다. 영양'교사', 아니 영양사가 학생들에게 실과를 가르치지 않습니다. 아니 그 어떤 것도 가르치지 않습니다. 올해 대통령님 취임 후에 초등, 중등 임용 티오가 초전박살 나는 동안 영양'교사' 티오는 폭발적으로 확대된 줄로 압니다. 학생들의 교육을 위해 정말 교원의 수를 줄이고, 영양 교사의 수를 늘리는 것이 올바른 해답인지 묻고 싶습니다. 마지막으로 교원 당 학생 수. 이 통계도 제대로 내주시기 바랍니다. 다른 나라들은 '정규 교원', 즉 담임이나 교과 교사들로만 통계를 내는데(너무나 당연한 것이지요) 왜 한국은 보건, 영양, 기간제, 영전강, 스강 까지 다 포함해서 교원 수 통계를 내는건가요? 이런 통계의 장난을 통해 OECD 평균 교원 당 학생 수와 거의 비슷한 수준에 이르렀다고 주장하시는건가요? 학교는 교육의 장이고 학생들의 공간이지, 인력 센터가 아닙니다. 부탁드립니다. 부디 넓은 안목으로 멀리 내다봐주시길 간곡히 부탁드립니다.
  

특정 문서와 유사한 문서 (DM 방식):
유사도 0.75, 문서 인덱스 96278:
현재 청와대 앞에서 비정규직 교사들의 정규직 전환에 대한 건의가 이루어지고 있는 것으로 알고 있습니다. 국민의 입장에서 그리고 교사 친구들과 비정규직 교사 친구들을 둔 친구의 입장에서 비정규직 공무원을 정규직으로 채용한 갓에 빗대어 현 상황을 해결하기엔 다소 무리가 있다고 생각되어 채용안을 건의 합니다. 현재 정규직 교사보다는 비정규직 교사가 더 많은 비중을 차지하고 그만큼 업무 비율 담당도 비정규직 교사들이 더 많이 하고 있는 것으로 알고 있습니다 왜냐면 학교에서 근무 및 생활을 잘해야 다음 기간제 교사에 또 재임용될 수 있기 때문입니다. 제 말은 비정규직 교사들의 마음도 이해가 간다는 것입니다. 하지만 현재 교직에 정규직으로 있는 수많은 교사들은 사립같은 경우를 제외하고는 기간제 교사 업무 중에 남는 시간과 여가 시간을 다 투자하여 정규직으로 임용된 교사들도 많습니다. 이 말은 즉, 현재 정규직 교사들의 노력을 헛투루 봐서는 안된다는 것입니다. 고로, 제가 건의하고자 하는 채용안은 현재 기준으로 기간제 교사 5년 이상인 자에 한하여(여기서 기간제 교사 기준을 5년으로 잡은 이유는 5년 정도는 한 분야에서 일을 해야 어느 정도 업무에 대한 파악도가 형성되어질 것이라고 생각하기 때문입니다) 필요한 합격자 수를 정하여 임용고시를 보는 것입니다. 즉, 그들만의 리그를 치루는 것입니다. 임용고시 수준은 현 임용고시 수준으로 하여 등수로 커트하는 것이고 지역 선택권은 없습니다. 지금 정규직 전환을 원하는 비정규직 교사들이 순수한 마음을 가지고 교직에 임하는 것이라면 근무 지역 선택이 그들에게 뭐가 중요하겠습니까. 그래서 그들을 필요한 곳에 배치시키면 될거 같습니다. 물론 위에서 말한 합격자 TO도 거기에 맞춰서 나오겠지요. 그리고 이 현안을 또한 장기적으로 본다면 이런 방식으로 교사를 뽑는 것이 더 합리적일 수 있다라는 생각이 듭니다. 외국 같은 경우에는 최소 석사 정도 되는 초중고 선생님들이 많은 반면 한국 같은 곳은 좋은 사범대 좋은 교육대 나와서 학사를 가지고 바로 현장으로 가서 아이들을 가르치고 생활합니다. 물론 교생 시스템을 반론으로 드실수도 있지만 교생 실습 기간은 실제 그들이 업무를 해야하는 기간에 비해서는 엄청 짧게 합니다. 가서도 실질적인 업무보다는 학생들과의 소통 정도로 끝나는 수준입니다. 그러므로 이 제도를 더 활성화시켜 좀 더 전문적인 교사들을 양성할 수 있고 그들이 교사라는 안정적인 직업만을 추구해 교사로써의 윤리적인 마인드는 가지지 않고 바로 선생님이 되는 것을 방지하며 한 개인으로써도 교사의 길이 맞는지 5년간 충분히 경험해 볼 수 있는 기회 또한 될 수 있을것이라고 생각합니다. 그래서 결론 맺자면, 기존 비정규직 교사들이 정규직 교사 전환을 원하는데 그에 따른 조건으로 현 5년차 이상 비정규직 교사 중 응시 희망자를 뽑아 지역 선택 없이 합격자 수를 부족한 TO대로 생성하여 임용고시를 현 임용고시 수준으로 보고 순위별로 합불합을 정해 정규직으로 전환 했으면 좋겠습니다. 그리고 나아가 이러한 방식으로 갈수록 아이들도 줄어드는데 좀 더 나은 교육의 발판을 마련함으로써 부모님들이 믿고 보낼수 있는 능력있고 경험있는 학교를 장기적으로 만들었으면 하는 생각입니다. PS. 이런 글을 처음 올려 두서가 없는 편이지만 관심 있으신분들은 충분히 이해하실수 있을거라 믿습니다 읽어주셔서 감사합니다! 좋은 하루들 되세요~
  

유사도 0.72, 문서 인덱스 14121:
국공립 학교의 기존 정교사들은 과원 발생시 다른 분야의 정교사2급자격증을 소지하면 임용시험 통과와는 무관하게 타 교과나 타 비교과로 전직(전과)이 가능합니다. 현재의 교직사회에도 수천명에 해당하는 기존의 정교사들이 이와 같은 방식으로 전직(전과)하여 전문성을 갖추지 못한 상태에서 버젓이 근무하고 있습니다. 전문성을 갖추지 못한 기존 정교사들의 타 교과 및 타 비교과로의 전직(전과)을 금지해 주기를 제안하는 바입니다. 지난 여름 우리 나라의 교직 사회에서는 기간제교사의 정규직화 문제로 많은 혼란이 있었습니다. 임용시험을 통과하지 못한 전문성이 결여된 기간제 교사들에게 자신들의 자녀를 맡길수 없다는 학부모 단체 및 형평성 문제를 거론한 교총과 많은 예비교사들의 외침의 여론을 수용하여 교육부는 기간제교사들의 정규직화를 무산시킨 바 있습니다. 최소 수년에서 최대 수십년간 정교사자격증을 가지고 있으면서 자신이 전공한 분야에서 근무하여 많은 현장 경력을 가지고 있는 기간제 교사들의 채용과정과 신규 예비교사들의 채용과정이 동일해야 하는가에 대해 많은 의문을 가졌으나 교육부에서 제시한 형평성 차원에서의 문제점에 대해 공감하여 문제 제기를 하지 않았습니다. 법적인 문제는 차치하고라도 기회는 평등하고 과정은 공정하고 결과는 정의로워야 한다고 생각하기 때문입니다. 하지만 기존 정교사들의 전직(전과) 실태는 형평성을 중시했던 교육부의 이중적인 잣대이기에 문제 제기를 합니다. 교육부 또는 시도 교육청은 과원 교사가 발생할 시 타 교과나 타 비교과의 정교사 자격증을 소지하고 있는 현직 정교사들에게 전직(전과)을 허용해 줍니다. 또한 단기 연수 과정을 개설하여 해당 분야의 정교사자격증을 빠르게 취득할 수 있도록 하는 행정 편의도 제공한 사례가 매우 많습니다. 하지만 이런 정교사들은 해당 분야의 현장 수업 및 업무 경력도 없으며, 더더욱 해당 분야의 임용시험을 통과하지 못한 상태이기에 전문성이 없는 상태입니다. 이런 정교사들에게 해당 분야의 업무를 맡긴다는 것은 교직사회가 스스로 전문직 문화라는 것을 내던지는 처사일 것입니다. 또한 이런 비전문가인 정교사들이 해당 분야의 자리를 차지하고 있다면 해당 분야를 전공하여 정교사자격증을 가지고 있으면서 수년째 임용시험을 준비하면서 전문적 지식을 연마하고 있는 많은 청년들에게 교육부가 스스로 형평성을 내던지는 처사일 것입니다 전문성을 갖추지 못한 기존 정교사들의 타 교과 및 타 비교과로의 전직(전과)을 금지해 주기 바랍니다. 또한 형평성을 고려하여 이미 전직(전과)하여 근무하고 있는 기존 정교사들도 재평가하여 전직(전과)을 재허용해 주어야 합니다. 평가방식은 교육부나 시도 교육청이 숙고해야 하겠으나 임용시험에 준하는 방식이어야 형평성에 적합할 것입니다. 단적인 예로써 정교사 경력이 있는 분들이기에 임용시험과정에 있는 교육학 시험과 구술 면접은 제외하더라도 임용 시험 전공분야의 통과 커트라인의 전국 평균이나 가장 낮은 시도의 통과 커트라인 이상으로 기준을 잡는 것과 타 교과로 전직(전과)이므로 당연히 수업 시현도 거쳐야 한다고 생각됩니다.(임용시험의 과락 점수를 기준으로 세우는 것은 너무 안일한 처사이기에 제외함) 전문성을 갖추어야 할 분야에 주먹구구식으로 기존 정교사들을 배치하여 형평성에도 어긋나고 전문직 사회인 교직사회 문화도 저해시키는 전직(전과) 행정은 금지되어야 합니다.

  
유사도 0.72, 문서 인덱스 61399:
교사의 성과를 무엇으로 평가해야 하는가? 서울대학을 많이 합격시킨 담임? 학급별 전문대를 포함한 대학교에 많이 진학시킨담임? 환경미화를 깨끗이한 담임? 1년간 학급이 무결석인 담임? ....... 과연 무엇이 평가의 기준인지 알수가 없습니다. 대학을 많이 진학한것이 담임의 역량이 우수하다는 것을 증명할 수 있읍니까? 환경미화를 잘하면 학생들의 인성이 좋아져서 학생들이 사회에 나가면 법을 잘지킵니까? 무엇을 위한 평가인지 궁금합니다. 매스컴을 통해 서울대합격자수가 높은 학교가 우수학교를 인식되고 있습니다. 그렇다면 서울대학교가 우수학교라는 것을 논문의 수로 평가합니까? 범죄자수가 적은 학교가 우수한 학교 아닌가요? 평가라는 것이 너무 어리석다는 생각을 하지 않습니까? 교원평가를 한다는 것은 우매한 행동이라 생각합니다. 따라서, 교원성과급은 폐지되어야 합니다. * 추신 : EBS수능 출제도 폐지되어야 합니다.  


유사도 0.72, 문서 인덱스 159175:
안녕하세요? 사범대를 졸업하고 기간제 교사를 하며 임용을 준비하고 있는 학생입니다. (현재는 그 기간제도 구하지 못해서 백수입니다.) 현재 중등교사 임용 시험의 경쟁률은 절망적입니다. 40:1은 기본이니까요. 이렇게 말하면 다른 직종의 사람들은 일반회사 입사 경쟁률도 그 정도 된다며 그냥 참으라고 말합니다. 하지만 사범대의 경우는 좀 많이 다릅니다. 전공이 교육으로 특화 되어 있다보니 임용시험에 떨어지고 나서 다른 회사로 입사하는 것이 거의 불가능합니다. 솔직히 어떤 일반회사가 국어교육과, 수학교육과 학생들을 신입사원으로 받아주겠습니까? 졸업하고 잘하면 기간제 교사(정교사가 아프거나 출산, 육아시 대체하는 교사)정도 인데, 이 마저도 정교사 티오 자체가 없어서 구하기 힘듦니다. (저도 작년에 12달 중 1달, 2달, 3달 쪼개서 총 6개월 구했습니다. ㅠ.ㅜ) 초등의 경우는 초등교육 전공의 특수성을 인정하고 국가에서 미리 수급을 조절합니다. 그래서 적체되어 있는 예비교사들이 많지 않습니다. 그런데 중등의 경우, 출산율이 낮아져 앞으로 교사 티오가 점점 줄어들 것을 알고 있음에도 불구하고 수급 조절을 위한 어떠한 노력도 하지 않습니다. 대부분의 중등 예비교사들은 출산율로 인해 티오가 주는 것은 납득하고 받아들이고 있습니다. 다들 힘들어서 결혼과 출산을 포기하고 있는 추세니까요. 하지만 이를 어느정도 대비하고 조절할 수 있음에도 국가는 유독 중등임용에 대해서는 아무런 노력도 하지 않는 것처럼 보입니다. 초등처럼 앞으로 필요한 교원의 수를 미리 예상해서 사범대 인원, 교직이수 인원, 교육대학원 인원을 과목별로 어느 정도 조절한다던지 할 수 있을 텐데, 전국의 사범대, 교육대학원과 교수들 눈치를 보느라 그런지 아무런 노력도 하지 않습니다. 덕분에 사범대, 교육대학원생, 교직이수생 97% 이상이 졸업과 동시에 백수가 됩니다. 단순 개인의 무능력으로 치부하기엔 너무 심하지 않은가요? 뭐든 해주세요. 중등에도 관심을 가져주세요... 많이 절실합니다. 그러나 앞으로도 낮은 출산율로 나아질 기미가 보이질 않아 더 절망적입니다. 설령 이 청원이 20만이 넘지 못한다 하더라도, 한번 쯤은 생각해 주시킬 바라는 마음에서 올립니다.
  

유사도 0.72, 문서 인덱스 373748:
공무원,시서교사,공기업 채용하지말아주세요. 육군,해군,공군,해병대 군무원 채용하지말아주세요. 운전직공무원채용하지말아주세요. 사서직공무원이랑,사서교사 채용하지말아주세요. 정말 도서관 사서랑 사서교사는 필요가없습니다. 특히 사서교사는 임용고시 통과한 국어교사가 하면 되는일이지 학교에 사서랑 사서공무원뽑는것은 정말 불필요한 행위입니다.예산낭비에 인력낭비입니다. 그돈으로 어려운서민이나 불우이웃 보건복지 국민건강보험 보장성에 신경을 써주십시오, 뽑더라도 정규직 사서랑 사서공무원은,사서교사 채용해주시지 마십시오. 운전직공무원도 제발좀 채용하지말아주십시오. 뽑더라도 표창,상장,무사고 경력이런게 아니라 정비자격증소지나 얼마나 오랫동안 버스회사 경력이 있는지를 판단해주십시오. 그리고 운전직공무원 국가직(경력채용)은 필기시험없던데, 이거 불합리 합니다. 경력채용이든 운전직이든 간호직은 필기시험 도입해주십시오. ncs도입해주십시오. 경력채용도 운전직공무원 필기시험도입해주십시오. 간호직두요. 그리고 여경 정말 많이뽑던데 여경 별 필요없습니다. 여경채용을 폐지해주십시오. 일반직공무원들도 갈수록 대규모 채용을 하고잇던데 채용하지말아주십시오. 많이 채용하지말아주십시오. 공무원,공기업 특성상 철밥통이라 한번 되면 짜를 수도 없는데 부디 재고해주십시오. 마이스터,특성화고 고즐특채,고졸전형 폐지해주십시오.대졸자역차별입니다. 지역인재도 폐지해주십시오, 수도권역차별입니다. 기능직,무기계약직 채용할때도 필기시험도입이나 경쟁해서 겨루게 해주십시오. 그냥 면접만으로 채용하거나 기간지낫다고 전환이 아니라 필기로 경쟁해서 겨루게 해주십시오. 전반적으로 지금 모든 군무원, 공무원,공기업은 넘쳐납니다. 더 뽑을 필요가 없습니다. 이번정권이 정말 공공기관 채용에 무기계약직 전환 정규직전환 정말 많은 데 다 폐지해주십시오. 일자리창충을 공공기관이 아닌 사기업으로 늘려주십시오. 모든 학교에 사서교사 및 사서공무원을 의무배치하는 것에 반대합니다. 학교에서 사서교사나 사서공무원이 할 일이 무엇일까요? 쉬는시간이나 점심시간에 도서관에서 도서대출해주는 일이 전부이지 않을까요? 학교독서진흥을 위해 도서관 활용수업 및 독서교육 수업을 의무화시키지 않는다면요. 단순히 학교도서관에서 편히 도서관만 지키며 사생활을 즐기며 국민 혈세로 월급받는 사서교사 및 사서의 의무배치(학창시절 이런 놀고먹는 국민혈세 축내는 사서교사,영양교사,보건교사 많이 봤습니다) 사서나 사서교사 뽑지말고 임용고시 합격한 국어교사에게 월급조금더 주고 그일을 시키는게 백번 맞습니다. 전국 각급 도서관에서도 사서의 역할은 미비하고 하는게 없습니다. 채용금지 및 채용규모를 줄여주십시오. 부탁드립니다. 공무원 특성상 한번 채용하면 정년이 보장되고 매년 호봉이 자동상승되어 월급과 제반복지가 올라가서 엄청난 인건비 부담이 국민세금인거 인지해주십시오. 모든직렬의 공무원 숫자를 많이 뽑을 필요가없습니다. 감소해주십시오.이제 4차산업혁명시대이고 무인의시대입니다. 교사는 학생들을 대상으로 수업을 하거나 담임을 하는 등 직접적인 교육활동을 하는 것이라 생각됩니다. 그런데 영양사와 사서는 왜 교사라는 명목으로 임용하는 것인지 도저히 이해가 되지 않습니다. 요즘 정부 일자리 대책이 도대체 어떠한 기조로 가고 있는 것인지 한심합니다. 교과교사는 뽑지 않고 사서, 영양과 같은 비교과에 대한 교사수급만 늘리니 학생 교육은 누가하고 학교 행정업무는 누가 합니까? 국민 세금만 축내고 하는 일 없이 학교에 있는 사람들이 교사입니까? 실제 학교 현장에 나와서 고민해 보고 정책 방향을 정하는 것인지 매우 의심스럽니다. 실제 학교 현장에서 목소리를 들어보고 정책에 반영을 하십시오. 현 정권의 많은 장점에도 불구하고, 이번 건 관련해서 조급증이 염려되어 한 말씀 올립니다. 1. 사서직을 임명했을 때 들어가는 월급과 유지비, 연금 등에 대한 국민적 합의가 있었는지요? 2. 일과 중엔 수업 듣기 바쁘고, 일과 후엔 학원으로 내빼는 아이들을 도서관에 잡아둘 수 있는지요? 3. 학부모 입장에서 '학과 공부말고', '한가하게' 독서하고 있는 자녀를 두고 볼 수 있는지요? 4. 모든 학교가 사서교사를 수용할 만큼의 장서와 시설을 갖추고 있는지요? 5. 현재 사서교사가 있는 학교에서 과연 사서교사의 필요성을 간절하게 느끼고 있는지 교장교감들께 은밀하 내사를 해 보셨는지요? 6. 현재 기간제로 근무하는 사서직 대체 인력은 어찌 되나요?국가기관, 지방자체단체 등 각종 공공기관에 임용되는 운전직공무원의 업무량은 기관별 특성에 따라 다르겠지만, 운전 및 차량관리 하나만을 본다면 그 업무는 다른 직종과 비교해서 너무도 편하다라고 할 수 있습니다. 본인이 소속된 기관의 경우 운전직공무원들의 월별 운행 횟수를 파악해보면 상당히 놀라울 정도입니다. 유관기관행사, 상급기관에서의 기관장회의 등을 제외하면 거의 운행이 없습니다. 또한, 운행을 나간다하여도 운행시간 외에 대기하는 시간이 상당하여 이건 엄청난 낭비다라는 생각이 들 정도입니다. 사실, 차관급 이상의 기관장에게만 전용기사(별정 비서직 ) 및 전용차량이 지원되도록 규정에 명시가 되어있는데, 그마저도 제대로 이행이 안 되고있다는 사실 또한 문제입니다. 하지만, 이 문제는 논외로 하고 운전직공무원들에 대해서만 얘기하고싶습니다. 운전직공무원을 지원한다는 수많은 수험생분들에게 그 이유를 물어보면 대부분이 업무량이 적고 편해서라고 합니다. 네, 맞습니다. 말도 안 될 정도로 편하지요. 그러나, 개념이 있는 운전직공무원 중 상당수는 각종 행정업무를 전담하여 처리하고있습니다. 아마도 행정업무 90%, 운전업무 10%정도로 업무를 본다고 평가할 수 있을 것 같습니다. 문제는 이러한 운전직공무원들은 극히 소수라는 점입니다. 운전직공무원, 이들이 변하지 않는다면 행정부에서 이러한 문제를 해결해줘야겠지요. 저는 두 가지 제안을 드리고싶습니다. 1. 기술직군의 운전직렬을 폐지하여주십시오. 2. 기존 운전직공무원들에게 규정에 명시된 행정업무 수행기간 등을 고려 타 직군이라 할지라도 전직의 기회를 부여하여 주십시오. ** 만약 제한이 된다면 이들이 운전만해도 된다는 생각 자체를 하지 않도록 운전직렬의 명칭을 개정해주시기를 부탁드립니다. (구체적인 방안 마련을 추진하고하 할지 참여의사도 있습니다.) 군무원,사서직,운전직,행정직,공기업 채용 대폭 감소시키고,여경채용 대폭 축소시키고 대학도 내신으로 가는 수시는 실력가늠도안되고 학교마다 시험문제 수준이 천차만별이라 불합리하니까 수능으로 대학가는 정시위주로 대학입학을 하게해주십시오. 제발 딱 못해도 10년만 저의 뜻을 받아주시기를 바랍니다. 딱10년만 제가 제안한대로 정책을 집행해주시기를 바랍니다. 정말 마지막민원이구요. 정말 진심으로 부탁드립니다.고졸로 공무원,공기업,대기업 입사해서 회사돈이나 사비로 다 나중에 대학가요. 그것도 재직자특별전형으로 명문대학교를 쉽게요. 제발 제가 제안한거 못해도10년동안만이라도 정책실현해주십시오. 익명보장 꼭 부탁드리구요. 전화는 하지말아주시고. 읽고 최대한 자세하고 꼼꼼히 논리적으로 답변달아주세요. 자세한 사항은 전화주십시오, 이런말씀하지말아주세요. 글이편해서 그래요. 제말 아시겟죠? 꼭좀 부탁드립니다. 정 뽑아야하면 기간제나 기간의 정함이 있는 계약직 공무직이런거로 뽑아주세요. 공무원이나 정규직은 제발 뽑지말아주세요....

  


